templates/ui-frameworks/vue/web-app/ui/src/ClientProvider.vue.hbs

```diff
@@ -0,0 +1,25 @@
+<template>
+  <progress v-if="loading"></progress>
+  <div v-else-if="error" class="alert">
+    Error connecting to Holochain: {{{{raw}}}} {{error.message}}.{{{{/raw}}}}
+  </div>
+  <slot v-else-if="client"></slot>
+
+  <button v-if="client && IS_HOLO" @click="signOut">Sign Out</button>
+</template>
+
+<script setup lang="ts">
+{{#if holo_enabled}}
+import WebSdk from "@holo-host/web-sdk"
+{{/if}}
+import { useClientProvider{{#if holo_enabled}}, IS_HOLO{{/if}} } from './clientProvider';
+
+const { client, error, loading } = useClientProvider();
+
+{{#if holo_enabled}}
+const signOut = async () => {
```

Comment:
We're always using `signOut` in the template, so either that should be conditional, or this function definition should not be.

Reply:
This functionality is already checked via the `holo_enabled` conditional. I added a missing conditional on the sign out button though.